stubs/untangle/untangle.pyi

```diff
@@ -0,0 +1,37 @@
+from _typeshed import Incomplete
+from collections.abc import Iterator
+from typing import Any
+from xml.sax import handler
+
+def is_string(x: Any) -> bool: ...
+
+class Element:
+    children: Incomplete
+    is_root: bool
+    cdata: str
+    def __init__(self, name: str | None, attributes: dict[str, Any] | None) -> None: ...
```

Comment:
Are you sure `attributes` can be `None`? It looks like `get_attribute()` will fail if `None` is passed in to that parameter: https://github.com/stchris/untangle/blob/e988ee2ce3cb296308ef0d1442c74bd4b17b3b1b/untangle.py#L65

Also: does it _have_ to be a `dict`, or could we use `Mapping` here? We generally prefer to be as permissive as possible for parameter annotations.

Reply:
I guess `attributes` _can_ be `None`, since the library literally does this in `Handler.__init__`: https://github.com/stchris/untangle/blob/e988ee2ce3cb296308ef0d1442c74bd4b17b3b1b/untangle.py#L143

Still seems weird to me though 🤷‍♂️